apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -224,6 +225,35 @@
                 <a class="btn btn-xs btn-secondary" href="<%= urlPrefix %>/assessment/<%= assessment.id %>/manual_grading/instance_question/<%= instance_question.id %>">Manual grading</a>
                 <% } %>
               </td>
+              <td class="text-right">
+                <div class="dropdown">
+                  <button type="button" class="btn btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    Action <span class="caret"></span>
+                  </button>
+
+                  <div class="dropdown-menu dropdown-menu-right">
+                    <% if (authz_data.has_course_instance_permission_edit) { %>
+                      <button
+                        id="row<%= i_instance_question %>PopoverBreakButton"
```

Comment:
Nit: any reason for this to use camel case instead of kebab-case?

Reply:
Code blindly copied from a three-year-old implementation 🙂 I'll tweak this.

Reply:
Actually, this was existing code that goes back to when Matt first wrote it. I'd prefer to leave it alone for now.